dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/CompanyManagerTest.kt

```diff
@@ -166,4 +135,21 @@ class CompanyManagerTest(
                 "$searchString were returned. At least one was expected.",
         )
     }
+
+    @Test
+    fun `check that it is not possible to get a result with no data type filter specified`() {
+        val exception = assertThrows<InvalidInputApiException> {
+            testCompanyManager.searchCompaniesAndGetApiModel(
+                StoredCompanySearchFilter(
+                    dataTypeFilter = emptyList(),
+                    countryCodeFilter = emptyList(),
+                    sectorFilter = emptyList(),
+                    searchString = "",
+                    nameOnlyFilter = false,
+                    uploaderIdFilter = emptyList(),
+                ),
+            )
+        }
+        assertEquals(exception.message, "Please specify a dataframework")
+    }
```

Comment:
This test is not required anymore after making `dataType` filter optional again, right?

Reply:
It's gone